code/modules/client/preferences_savefile.dm

```diff
@@ -104,6 +104,10 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 						new_value = JP_LOW
 			if(new_value)
 				job_preferences[initial(J.title)] = new_value
+	if(current_version < 23)
+		all_quirks -= "Physically Obstructive"
```

Comment:
Is this hygiene-related, or where is it coming from?

Reply:
I asked to include it here, so we don't need two preference versions, this is already broken on master.